frontend/src/api/usePutData.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from 'react';
+
+/**
+ * Custom hook updating data.
+ * @param {string} url - adress of request
```

Comment:
typo - address. You can add example address as a localhost so one can copy paste it 
eg. 
@param {string} url - address of request eg. 'http:localhost:8080/api/'